### PR TITLE
refactor: unlock page design

### LINKED ISSUE #2199: Update lock screen to match new onboarding
Update unlock screen to match new design:
https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%8FSpec?node-id=2227%3A12789

image

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/1781059196).

This PR updates the unlock page to match new designs/illustrations.

*Note: I don't see a design for full page view so I improvised.

Screen Shot 2022-02-01 at 1 36 41 PM

cc/ @kyranjamie @beguene @He1DAr

### CODE REVIEW

**Comment by Eshwari007** [comment]
@fbwoolf while testing this PR noticed that the send form is trucated
Screen Shot 2022-02-01 at 4 31 15 PM

**Comment by Eshwari007** [comment]
This seems to occur to an existing account,new accounts are looking fine
Screen Shot 2022-02-01 at 4 33 19 PM

**Comment by fbwoolf** [comment]
> @fbwoolf while testing this PR noticed that the send form is trucated

Interesting! I see that too. Maybe a merge conflict or something with all the style changes. I'll get it fixed and push to this branch so we can merge it into dev before releasing. Thx.

**Comment by fbwoolf** [comment]
@Eshwari007 pushed up a fix for the send form if you want to check it on your end when it's done building:

Screen Shot 2022-02-01 at 4 30 21 PM

File: src/app/pages/onboarding/sign-in/sign-in.tsx [hunk 1]
```diff
@@ -39,10 +39,10 @@ export const SignIn = () => {
         ) : (
           <>
             <Title as="h1">Sign in with Secret Key</Title>
-            <Caption>
+            <Text color={color('text-caption')}>
```
Comment (kyranjamie) [inline src/app/pages/onboarding/sign-in/sign-in.tsx:42]: What prompted this change?

File: src/app/pages/unlock.tsx [hunk 2]
```diff
@@ -112,8 +119,14 @@ export function Unlock(): JSX.Element {
           isLoading={loading}
           onClick={submit}
         >
-          Unlock
+          Continue
         </PrimaryButton>
+        <Caption textAlign="left">
+          Forgot your password? Unlock your account by{' '}
+          <Link display="inline" fontSize={-1} onClick={() => navigate(RouteUrls.SignIn)}>
```
Comment (kyranjamie) [inline src/app/pages/unlock.tsx:126]: is `-1` a legit value?

**Review by kyranjamie — APPROVED** [review]

**Comment by kyranjamie** [comment]
Merging this as I want the send form fix.

File: src/app/pages/unlock.tsx (hunk 2 above)
Comment (fbwoolf) [reply src/app/pages/unlock.tsx:126]: I am going to confirm this, I haven't worked with these number increments before and tbh was experimenting with it. It did work for 14px but I should confirm it in some docs.

**Pull request merged**